index.php

```diff
@@ -27,70 +27,83 @@
                                 <li><b>Catalogue reference(s)</b>. You can make a note of these by searching the catalogue in advance, or by searching from this page. You can order up to 12 documents.</li>
                             </ul>
                             <form action="" id="" class="" method="POST" novalidate="novalidate">
-                                <p>Fields marked <span class="mandatory">&#42;</span> are compulsory</p>
                                 <fieldset>
                                     <legend>Your reader's ticket</legend>
                                     <div class="form-row">
-                                        <label for="readers_ticket">Reader's ticket number <span class="mandatory">&#42;</span></label>
+                                        <label for="readers_ticket">Reader's ticket number</label>
                                         <input type="text" id="readers_ticket" name="readers_ticket" class="small" required>
                                     </div>
                                 </fieldset>
                                 <fieldset>
                                     <legend>Your documents</legend>
                                     <div class="form-row">
-                                        <label for="date_of_visit">Date of visit <span class="mandatory">&#42;</span></label>
-                                        <input type="text" id="date_of_visit" name="date_of_visit">
+                                        <p id="dob"><b>Date of visit</b></p>
+                                        <span id="reference-1-error" class="form-error form-hint">Please insert a valid day</span>
+                                        <div id="groupErrorPlacement">
+                                            <span class="dob-col day">
+                                                <label id="day" for="certificate_day">Day</label>
+                                                <input type="number" aria-labelledby="dob day" id="certificate_day" name="certificate-day" class="form-warning">
```

Comment:
Are these fields required? If so, please use inclusive of 'required':

aria-required="true"

And add this att if there are errors:

 aria-describedby="error-message-id"